tests/functional/common.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+import grpc
+import os
+import tritonclient.grpc as grpcclient
+from tritonclient.grpc import service_pb2_grpc
+
+def prepare_inputs(input_name, shape):
+    inputs = []
+    inputs.append(grpcclient.InferInput(input_name, shape, "FP32"))
+
+    inputs[0].set_data_from_numpy(np.ones(shape, dtype=np.float32))
```

Comment:
Consider using these methods to convert datatypes

from tritonclient.utils import np_to_triton_dtype, triton_to_np_dtype